README.md

```diff
@@ -1,14 +1,37 @@
 # Auth Info Dumper
 
-This is a PS4 payload to dump [auth info](https://www.psdevwiki.com/ps4/Auth_Info) from a SELF file. Right now it is hard-coded file name to dump and it is only work on 9.0.
+This is a PS4 payload to dump [auth info](https://www.psdevwiki.com/ps4/Auth_Info) from a ELF, SELF, or eboot.bin file. This payload only works on firmware 9.00!
+
+## How to use
+
+1. Download .bin from [Releases](https://github.com/obhq/auth-dumper/releases/latest) or build it yourself!
+2. Setup your prefered payload loader, for Payload Guest, put the payload on the root of your usb under a folder named `payloads` or on your PS4 under `/data/payloads`
+3. On the root of your USB, make a file named `dump-list.txt`
+4. This payload can dump multiple auth-infos on one execution, so open `dump-list.txt` and format each line like this: `/readauthinfofrom/folder/file.self:/exportauthinfoto/file.bin` An example is below:
+```
+/system/sys/SceSysCore.elf:/data/dumpedSysCore.bin
+/system/sys/SceVdecProxy.elf:/data/dumpedVdecProxy.bin
+```
```

Comment:
This format broke the rendered list.

Reply:
Uh, no? That two line list was meant to be in a code block... Which it is.

Reply:
![Screenshot 2023-12-08 at 23-21-54 auth-dumper_README md at 9df164978d22f5dd22be63dceb2f82e247b3c52c · obhq_auth-dumper](https://github.com/obhq/auth-dumper/assets/3048307/0e7f5d20-0c5c-49d2-bd23-224f5d18c01c)

Reply:
There, that should fix it.